notificationbar/schema.json

```diff
@@ -96,13 +101,21 @@
                 "type": "string",
                 "pattern": "^(normal|bold(er)?|lighter|1000|([1-9]{1}[0-9]{0,2}))$"
               },
-              "^(background-)?color$": {
+              "^background-color$": {
```

Comment:
Could you give some details for the changes made to the pattern of the allowed css here and below?

Reply:
I clones the pattern from here to color, so i think we are good.